docs/design/ros_workspace.md

```diff
@@ -25,20 +25,20 @@ subfolders.
 
 Additionally, this principle is meant to describe what goes in a package. Generally
 speaking, a package should implement either a single ROS node, a collection of
-like-nodes, or define shared utilities/helpers that are used by other packages. For
+like-nodes (often only one of which is launched at a time), or define shared utilities/helpers that are used by other packages. For
 instance, the [`launch_utils`](./../../workspace/src/common/launch/launch_utils/) package
 does not have a node, but implements utilities used by other launch files.
 
 ### Principle 2: Metapackages and launch files organize vehicle spin up/tear down
 
 It is certainly possible that there exists multiple ART vehicles each with a different
-setup (i.e. different sensors, computational hardware, etc.). Therefore, this principle
+setup (i.e. different sensors, computational hardware, mission tasks etc.). Therefore, this principle
 helps to define which nodes are created and/or built as it depends on the specific
 vehicle platform in use.
 
 First, [metapackages](https://wiki.ros.org/Metapackages) are a new-ish ROS construct which helps define the build dependencies for a specific package. Essentially, a metapackage has no nodes or code. It is an empty package except for a `package.xml` and `CMakeLists.txt` file which define build dependencies. These build dependencies can then be used to directly build nodes/packages for a specific vehicle platform by only using `colcon build` to build that package.
 
-For instance, if a hypothetical vehicle platform requires packages named `camera_driver`, `lidar_driver`, `perception`, `control`, and `actuation`, you can specify all these packages as `<exec_depend>` in the metapackage. When `colcon build --packages-up-to <metapackage>` is run, the `<exec_depend>` packages are automatically built.
+For instance, if a hypothetical vehicle platform requires packages named `art_launch`, `object_detection`, `centerline_objects_path_planner`, and `pid_lateral_controller`, you can specify all these packages as `<exec_depend>` in the metapackage. When `colcon build --packages-up-to <metapackage>` is run, the `<exec_depend>` packages are automatically built.
```

Comment:
this may age poorly, is it really confusing to have it as before where they aren't actual package names?

Reply:
I found it confusing when it wasn't actual package names - my thought process was "okay but what is an ACTUAL example of what I am supposed to put here?".